copy_dev_postgres_to_v14.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh -e
+target_namespace="hmpps-interventions-dev"
+
+echo "Going to stop and migrate $target_namespace"
+echo "Please verify             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ and abort if necessary."
+sleep 2
+
+
+echo
+echo "✨ Stopping UI"
+kubectl scale \
+  --namespace="$target_namespace" \
+  --replicas=0 \
+  --timeout=5m \
+  deployment/hmpps-interventions-ui
```

Comment:
"Scale to 0 replicas" means no instance of the UI application will be running, effectively cutting off the service from end users.

All in-progress requests can complete within the grace period (10 seconds), which is usually more than enough for all requests to complete.